src/gatsby-plugin-chakra-ui/theme.js

```diff
@@ -130,6 +130,13 @@ const customTheme = {
       textTransform: 'uppercase',
     },
   },
+  footer: {
+    background: '#001514',
+    iconsBackground: '#C4C4C4',
+    iconsFill: '#000000',
+    text: '#F7F7F2',
```

Comment:
We can use the `rbb-white` variable instead of redefining this here!

Reply:
Yep, missed that one too!